blackmarket/UI/ViewModels/SignUpViewModel.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  SignUpViewModel.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 9/30/22.
+//
+
+import SwiftUI
+
+class SignUpViewModel: ObservableObject, Identifiable {
+    
+    private let dataPrivacy = "https://www.osthus.com/osthus-glossary/data-policy/#:~:text=A%20data%20policy%20contains%20a,data%20quality%2C%20and%20data%20architecture."
+    
+    private let cockiesPrivacy = "https://www.grafterr.com/cookie-policy?gclid=CjwKCAjwp9qZBhBkEiwAsYFsb5_jgSsJFbQvqYDNyqLhOrgi8KZZqZjcZjUXKk9-Cj-vbV0ZgQgx6RoCIKEQAvD_BwE"
+    
+    @Published
+    var emailConfiguration = TextFieldViewConfiguration(
+        title: LocalizedString.SignUpTextField.emailTitle,
+        placeholder: LocalizedString.SignUpTextField.emailPlaceholder,
+        validations: [.email, .nonEmpty],
+        errorMessage: LocalizedString.SignUpTextField.emailError
+    )
+    
+    @Published
+    var nameConfiguration = TextFieldViewConfiguration(
+        title: LocalizedString.SignUpTextField.nameTitle,
+        placeholder: LocalizedString.SignUpTextField.namePlaceholder,
+        validations: [.nonEmpty],
+        errorMessage: LocalizedString.SignUpTextField.nameError
+    )
+    
+    @Published
+    var passwordConfiguration = TextFieldViewConfiguration(
+        title: LocalizedString.SignUpTextField.passwordTitle,
+        placeholder: LocalizedString.SignUpTextField.passwordPlaceholder,
+        validations: [.nonEmpty],
+        isSecure: true,
+        errorMessage: LocalizedString.SignUpTextField.passwordError
+    )
+        
+    var isValid: Bool {
+        return [emailConfiguration, passwordConfiguration].allSatisfy { $0.isValid }
+    }
+    
+    lazy var disclaimerString: AttributedString = {
+        do {
+            var text = try AttributedString(markdown: "\(LocalizedString.SignUpView.disclaimer) [\(LocalizedString.SignUpView.dataPolicy)](\(dataPrivacy)) \(LocalizedString.SignUpView.disclaimerEnd) [\(LocalizedString.SignUpView.cookiesPolicy)](\(cockiesPrivacy))")
+     
+            if let range = text.range(of: LocalizedString.SignUpView.dataPolicy) {
+                text[range].font = .subheadline.bold()
+            }
+            
+            if let range = text.range(of: LocalizedString.SignUpView.cookiesPolicy) {
+                text[range].font = .subheadline.bold()
+            }
```

Comment:
```suggestion
            let dataPolicyString = LocalizedString.SignUpView.dataPolicy
            let cookiesPolicyString = LocalizedString.SignUpView. cookiesPolicy
            
            var text = try AttributedString(
                markdown: """
                \(LocalizedString.SignUpView.disclaimer) [\(dataPolicyString)](\(dataPrivacy)) \(LocalizedString.SignUpView.disclaimerEnd) [\(cookiesPolicyString)](\(cockiesPrivacy))
                """
            )
            
            let highlights = [dataPolicyString, cookiesPolicyString]
            for highlight in highlights {
                if let range = text.range(of: highlight) {
                    text[range].font = .subheadline.bold()
                }
            }
```

You could probably break the long line even more